docs/changelog.rst

```diff
@@ -26,6 +26,23 @@ KSQL 5.3.0 includes new features, including:
 * ``CREATE STREAM`` and ``CREATE TABLE`` will now allow you to create the topic if it is missing.
   To do this, specify the ``PARTITIONS`` and optionally ``REPLICAS`` in the ``WITH`` clause.
 
+* KSQL now supports deserializing records where the value is:
```

Comment:
The first two bullets don't seem related to this PR. Perhaps they were accidentally picked up in a bad merge?

Reply:
yes. seems like a bad merge indeed